wandb/cli/cli.py

```diff
@@ -222,7 +222,12 @@ def projects(entity, display=True):
     "--relogin", default=None, is_flag=True, help="Force relogin if already logged in."
 )
 @click.option("--anonymously", default=False, is_flag=True, help="Log in anonymously")
-@click.option("--verify", default=False, is_flag=True, help="Verify login credentials")
+@click.option(
+    "--verify/--no-verify",
+    default=False,
+    is_flag=True,
+    help="Skip verifification of login credentials",
```

Comment:
typo ("verifification")

since the default at the moment is to skip, let's keep this as "Verify login credentials"